pom.xml

```diff
@@ -45,11 +44,6 @@
       <artifactId>gson</artifactId>
       <version>2.8.6</version>
     </dependency>
-    <dependency>
```

Comment:
do we not need this ?

Reply:
We do, I was deleting duplicate dependencies from when I merged and I accidentally deleted a couple others.